src/main/webapp/app/overview/course-overview.scss

```diff
@@ -179,6 +179,12 @@ canvas#complete-chart {
     animation-iteration-count: initial;
 }
 
+@media (max-width: 500px) {
```

Comment:
Are we intentionally assuming here that sidebar is hidden? When sidebar is visible, overall width goes beyond `500px`.
Considering that sidebar takes around `300px`, I'd set `max-width: 800px`

Reply:
I originally wanted to encourage users to collapsing the sidebar, but 500px was inaccurate, Ive tested it again and the correct width-border is 655px if we want to make it scrollable even if the sidebar is open. Anything above the 655px can be displayed properly and is already being handled by shifting elements, and the scroll becomes redundant. I'd therefore propose using 655px as the new border.